.github/workflows/tests.yml

```diff
@@ -17,28 +17,28 @@ jobs:
     name: "Unit Test"
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/cache@v2
-      id: npm-cache
-      with:
-        path: '**/node_modules'
-        key: ${{ runner.os }}-modules-${{ hashFiles('**/package-lock.json') }}-${{ secrets.NPM_CACHE_VERSION }} # increment NPM_CACHE_VERSION secret to force cache reset
-    - name: Install dependencies
-      if: steps.npm-cache.outputs.cache-hit != 'true'
-      run: npm ci
-    - name: Build
-      run: npm run build
-    - uses: saucelabs/sauce-connect-action@v2.2.1
-      with:
-        username: ${{ secrets.SAUCE_USERNAME }}
-        accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
-        tunnelName: $GITHUB_JOB
-    - name: Run Unit Test
-      run: npx karma start karma.saucelabs.conf.js --single-run
-      env:
-        BUILD_NUMBER: $GITHUB_RUN_NUMBER
-        BUILD_ID: $GITHUB_RUN_ID
-        JOB_NUMBER: $GITHUB_JOB
+      - uses: actions/checkout@v2.4.2
```

Comment:
(non-blocking nitpick) I would prefer to see this pinned to just v2. As-is, it requires manual work on our part to update the action.